src/Form/FormAnalyzer.js

```diff
@@ -177,9 +183,10 @@ class FormAnalyzer {
     }
 
     evaluatePageHeadings () {
-        const headings = document.querySelectorAll('h1, h2, h3, [class*="title"], [id*="title"]')
-        headings.forEach(({textContent}) => {
-            textContent = removeExcessWhitespace(textContent || '')
+        const headings = document.querySelectorAll('h1, h2, h3')
```

Comment:
Removed `[class*="title"], [id*="title"]` to avoid catching a bunch of useless stuff.